### ISSUE TITLE
Add a dyslexic ruler option.

### ISSUE THREAD

**Issue by developer-diganta** [developer-diganta/Dino#66]
### Subject of the issue
Add a dysexic ruler. It would ressemble a ruler and can be moved with the mouse. This would help readers concentrate on each line at a time. Check the image for reference.
![image](https://user-images.githubusercontent.com/65999534/211192428-c0a3f3b3-b371-45e3-be7a-a5249fb241af.png)

**Comment by Ria-R** [comment]
Hi!
I would like to work on this issue but would need some help as to how to begin with it. Is that fine?

**Comment by developer-diganta** [comment]
Yeah sure!  The flow for the extension is like this. The popup.html has a link to background.js . So when you click anything on popup.html (the extension essentially), the response can be got in background.js The background.js then sends the info to content.js. content.js can be used to manipulate the main web page.
So now you can add the option for the ruler in popup.html and add that via the js files to the webpage. The ruler can be a simple full width, some height, grayish colored div that moves with the mouse cursor.
